src/features/account/AccountProfileForm.tsx

```diff
@@ -26,9 +25,7 @@ export const AccountProfileForm = () => {
     refetchOnWindowFocus: false,
   });
 
-  const accountAvatar = useFetchAvatar(account.data?.image || '', {
-    enabled: !!account?.data?.image,
-  });
+  const accountAvatar = useAvatarFetch(account.data?.image || '');
```

Comment:
```suggestion
  const accountAvatar = useAvatarFetch(account.data?.image);
```

I think it should be ok like that

Reply:
useAvatarFetch requires the url to exist, so if we remove that, we have to modify the useQuery called inside of the hook